### PR TITLE
refresh gradle (to 6.0.1)

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,103 @@
...
+This project uses [flyway](https://flywaydb.org/getstarted). Gradle has been configured to wrap flyway, 
+exposing the relevant functionality as tasks. The tasks are only available if the database credentials
+have been defined; these default to reasonable values for developers, but it is a good practice to 
+explicitly specify them when invoking the tasks.
 
+It is highly recommended to install gdub (https://github.com/dougborg/gdub) because it handles some shortcomings 
+of gradle's commandline behavior. The instructions assume this, using `gw` instead of `./gradlew` of `gradle`.
+
+### MySQL
+MySQL scripts are compatible with MySQL 5.6.
```
Comment (malaffoon) [inline README.md:26]: I put the full MySQL instructions here in this project, and i'll reference it from the other three projects.

File: gradle.properties [hunk 2]
```diff
@@ -1,3 +1,9 @@
+version=2.4.0-SNAPSHOT
```
Comment (malaffoon) [inline gradle.properties:1]: Making this consistent with TIMS.

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,6 +1,6 @@
-#Mon May 01 14:03:20 PDT 2017
+#Mon May 04 08:40:24 PDT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0.1-all.zip
```
Comment (malaffoon) [inline gradle/wrapper/gradle-wrapper.properties:6]: Main point of this PR.

File: README.md [hunk 4]
```diff
@@ -1,44 +1,103 @@
...
+store ("reporting_olap"). For developers, there are test schemas for warehouse and reporting (because the OLAP
+datastore cannot be run locally, there is no test schema for reporting_olap). 
+
+This project uses [flyway](https://flywaydb.org/getstarted). Gradle has been configured to wrap flyway, 
+exposing the relevant functionality as tasks. The tasks are only available if the database credentials
+have been defined; these default to reasonable values for developers, but it is a good practice to 
+explicitly specify them when invoking the tasks.
 
+It is highly recommended to install gdub (https://github.com/dougborg/gdub) because it handles some shortcomings 
+of gradle's commandline behavior. The instructions assume this, using `gw` instead of `./gradlew` of `gradle`.
```
Comment (gregcharles) [inline README.md:23]: Nit: .gradlew or .gradle?

Comment (malaffoon) [reply README.md:23]: I think it's correct as is: most people either use the local script, `./gradlew` (or i suppose `./gradlew.bat` or their globally installed `gradle`.

**Pull request merged**